docs/guides/networking/linode-network/ip-sharing-atlanta/index.md

```diff
@@ -0,0 +1,265 @@
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+published: 2020-06-02
+modified: 2021-11-05
+modified_by:
+  name: Linode
+title: "Enable IP Sharing (Elastic IPs) in Atlanta through FRR"
+noindex: true
+_build:
+  list: false
+contributor:
+  name: Linode
+external_resources:
+- '[FRRouting Documentation](http://docs.frrouting.org/en/latest/overview.html)'
+aliases: ['/platform/manager/manually-enable-elastic-ip-on-your-linode/','/guides/manually-enable-elastic-ip-on-your-linode/']
+---
+
+The Atlanta data center was upgraded in April of 2021 to improve performance and expand product availability (see the [Linode Atlanta Data Center Upgrades Completed](https://www.linode.com/blog/linode/linode-atlanta-data-center-upgrades-completed/) blog post). As part of this upgrade, the [IP Sharing](/docs/guides/remote-access/#configuring-ip-sharing) feature was impacted and no longer functions as it did before. Customers that currently use this feature can follow this guide to manually enable IP Sharing (also called *Elastic IPs*) through the open source [FRRouting (FRR)](http://docs.frrouting.org/en/latest/overview.html#about-frr) tool. This allows two Linode Compute Instances to share a single IP address, one serving as the primary and one serving as the secondary. If the primary Compute Instance becomes unavailable, the elastic IP will seamlessly failover to the secondary Compute Instance.
+
+{{< note >}}
+This guide discusses a temporary workaround specific to the Atlanta data center. Future planned network upgrades may impact this feature again and may require additional configuration.
+{{</ note >}}
+
+## Before You Begin
+
+1. Prior to beginning the process outlined in this guide, make sure that you have received an IPv4 address(es) from Linode Support to use as your Elastic IP(s). To request an additional IPv4 address, [open a new support ticket from the Cloud Manager](/docs/guides/support/#contacting-linode-support).
+
+1. Ensure you have set the [hostname](/docs/getting-started/#set-the-hostname) and have updated the [hosts file](/docs/getting-started/#update-your-system-s-hosts-file) on your Compute Instances.
+
+1. Ensure Python 3 is installed on your system. See [FRR's official documentation](http://docs.frrouting.org/en/latest/installation.html#python-dependency-documentation-and-tests) to learn about FRR's Python dependencies.
+
+1. [Disable Network Helper](/docs/platform/network-helper/#single-per-linode) on the Elastic IP Linodes and reboot them.
+
+## Install FRR
+
+This section provides instructions for installing FRR on Debian, Ubuntu, and CentOS systems through their native package managers. If you're using a different distribution or prefer to install FRR from source, follow [FRR's official installation instructions](http://docs.frrouting.org/en/latest/installation.html) to install FRR using git.
+
+### Ubuntu and Debian
+
+**Supported distributions:** *Ubuntu 20.04, 18.04, and 16.04 | Debian 11, 10, and 9*
```

Comment:
@wildmanonline not all of these support distributions are given instructions in the Configure FRR section (Ubuntu 16.04, Debian 11 and 9). Should we consider pruning the list of supported distributions here? Or tell the reader that configuration instructions for those distros are outside the scope of the guide?